components/resume/editors/shared/DateRangeFields.tsx

```
@@ -170,6 +190,8 @@ const DateRangeFields = ({
 					<FormItem className="flex flex-row items-start space-x-3 space-y-0">
 						<FormControl>
 							<Checkbox
+								checked={isCurrentLocal}
```

Biome (1.9.4)

[error] This JSX property is assigned multiple times.

This attribute is assigned again here.


(lint/suspicious/noDuplicateJsxProps)